PiwikTracker.php

```diff
@@ -1288,6 +1288,30 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param string $proxy IP as string, for example "173.234.92.107"
+     * @param int $proxyPort
+     */
+    public function setProxy($proxy, $proxyPort = 80)
+    {
+        $this->proxy = $proxy;
+        $this->proxyPort = $proxyPort;
+    }
+
+    /**
+     * If the proxy IP and the proxy port have been set, with the setProxy() function
+     * returns a string, like "173.234.92.107:80"
+     */
+    public function getProxy()
+    {
+        if (isset($this->proxy) && isset($this->proxyPort)) {
+            return $this->proxy.":".$this->proxyPort;
```

Comment:
Can you maybe add some whitespace before and after the dots? That would be great :) Not super important though as PR looks good in general